src/sql/src/plan/statement/dml.rs

```diff
@@ -347,7 +347,7 @@ pub fn plan_explain_plan(
                 raw_plan,
                 row_set_finishing,
                 desc,
-                broken,
+                broken: true,
```

Comment:
Can we revert this? The `BROKEN` keyword has other implications downstream and is really something that should be picked up from the concrete syntax.

I think for this PR you just want to change `OptimizerTrace::new` as follows:

```rust
pub fn new(path: Option<&'static str>) -> OptimizerTrace {
    let subscriber = Arc::clone(&mz_ore::tracing::GLOBAL_SUBSCRIBER.get().unwrap())
        // Collect `explain_plan` types that are not used in the regular explain
        // path, but are useful when instrumenting code for debugging purpuses.
        .with(PlanTrace::<String>::new(path))
        .with(PlanTrace::<HirScalarExpr>::new(path))
        .with(PlanTrace::<MirScalarExpr>::new(path))
        // Collect `explain_plan` types that are used in the regular explain path.
        .with(PlanTrace::<HirRelationExpr>::new(path))
        .with(PlanTrace::<MirRelationExpr>::new(path))
        .with(PlanTrace::<DataflowDescription<OptimizedMirRelationExpr>>::new(path))
        .with(PlanTrace::<DataflowDescription<Plan>>::new(path));

    OptimizerTrace(dispatcher::Dispatch::new(subscriber))
}
```